blueprint/core/blueprint_ImageView.h

```diff
@@ -34,7 +34,7 @@ namespace blueprint
             {
                 juce::String source = value.toString();
 
-                if (source.startsWith("data:"))
+                if (source.startsWith("data:image/"))
```

Comment:
`juce::URL` class cannot handle data url properly because data-url doesn't contain `://` inside the url.
we can use `juce::URL` after this if-clause for `file://` or `http(s)://` protocol

Reply:
Urgh, that's annoying. I would love to see JUCE's URL class adhere to spec, maybe I'll open a bug with them